### PR TITLE
Add center crosshair to the OSD helper lines

### PR DESCRIPTION
**Pull request by McGiverGim**
To help centering elements, this PR adds a crosshair in the center of the OSD when moving elements:
image

image

I have added it with CSS only, maybe is better to use some SVG or similar, but for a first version is enough. After feedback from users, maybe is better to add some vertical/horizontal lines too to help with the border elements.

### CODE REVIEW

**Comment by McGiverGim** [comment]
Pushed a change to disable the pointer events over the crosshair. They were interfering with the drag and drop of the OSD elements.

**Comment by haslinghuis** [comment]
@McGiverGim 

Finally debugging android:
- made one little change for OSD HD on Android - maybe you can add it here.

image

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
